src/mp-instance.ts

```diff
@@ -1406,20 +1417,15 @@ function completeSDKInitialization(apiKey, config, mpInstance) {
             const roktOptions: IRoktManagerOptions = {
                 sandbox: config.isDevelopmentMode,
             };
-            mpInstance._RoktManager.init(roktConfig, roktFilteredUser, currentUser, roktOptions);
+            mpInstance._RoktManager.init(
+                roktConfig,
+                roktFilteredUser,
+                mpInstance.Identity,
+                mpInstance.Logger,
```

Comment:
This may be a topic for future discussion but I was thinking we could potentially pass in services as an object, similar to the other config options.

So this would theoretically become:

```
const services = {
    identityService: mpInstance.Identity,
    loggerService: mpInstance.Logger,
};

mpInstance._RoktManager.init(
    roktConfig,
    roktFilteredUser,
    services,
    roktOptions,
)
```

Reply:
I like this idea.  I'm making a tech debt ticket - https://mparticle-eng.atlassian.net/browse/SQDSDKS-7339